File: rock_paper_scissors.rb
while true do
  print "Player 1: choose rock, paper, or scissors! "
  player_1 = gets.chomp
  print "Player 2: choose rock, paper, or scissors! "
  player_2 = gets.chomp
  if player_1 == player_2
    puts "It's a draw"
  elsif player_1 == "rock" && player_2 == "scissors"
    puts "Player 1 wins!"
  elsif player_1 == "paper" && player_2 == "rock"
    puts "Player 1 wins!"
  elsif player_1 == "scissors" && player_2 == "paper"
    puts "Player 1 wins!"
  else
    puts "Player 2 wins!"
  end
  break
end

File: javascript/jasmine-standalone-3-2/src/RockPaperScissors.js
'use strict';

function RockPaperScissors() {
  this.player2 = "Computer"
}
// RockPaperScissors.prototype.getPlayer1 = function() {
//   return this.Player1;
// };
RockPaperScissors.prototype.play = function() {
var compare = function(player1,player2){
    if(player1===player2){
        return "No one wins";
      }
  if (player1==="Paper"){
    if(player2==="Rock") {
    return "Paper wins";
  }
  else{
    return "Scissors wins";
 }
}
  if(player1==="Scissors") {
    if(player2==="Rock") {
    return "Rock wins";
  }
    else{
    return "Scissors wins";
  }
}
if(player1==="Rock") {
  if(player2==="Paper") {
  return "Paper wins";
}
  else{
  return "Rock wins";
}
}
}
};
